src/docs/modules.html

```diff
@@ -89,7 +89,7 @@ <h2>Accordion</h2>
                     <section class="qg-accordion">
                         <article>
                             <input id="without-icon" type="checkbox" name="tabs">
-                            <h3 class="acc-heading"><label for="without-icon">Without icon <span class="arrow"> <i></i></span></label></h3>
+                            <h3 class="acc-heading"><label for="without-icon">Without icon <p class="subtitle">Subtitle can go here (optional)</p><span class="arrow"> <i></i></span></label></h3>
```

Comment:
Incorrect HTML placement. By default block level elements are not allowed inside the inline elements. This will throw error on HTML validation  

2 options 
1.) Change p to span 
2.) Add display block to label or display inline to subtitle